dockerfiles/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:latest
```

Comment:
This line can be replaced with `osuosl/django-centos:latest`. https://hub.docker.com/r/osuosl/django-centos/